ChangeLog.md

```diff
@@ -18,6 +18,14 @@
    hull computation on both side of a point sequence.
    (Bertrand Kerautret, [#1073](https://github.com/DGtal-team/DGtal/pull/1073))
 
+ - LogScaleProfile: new class to represent a (multi)scale profile e.g. a sequence
+   of statistics on digital lengths parameterized by a scale.
```

Comment:
yes, even if with the thickness notion of Meaningful thickness it is not directly a grid size but almost equivalent